plutus-tx/src/PlutusTx/Blueprint/TH.hs

```diff
@@ -117,3 +120,33 @@ mkSchemaClause ts ctorIndexes =
   mkSchemaConstructor (TH.ConstructorInfo{..}, info, naturalToInteger -> ctorIndex) = do
     fields <- for constructorFields $ \t -> [|definitionRef @($(pure t)) @($(pure ts))|]
     [|SchemaConstructor info (MkConstructorSchema ctorIndex $(pure (TH.ListE fields)))|]
+
+deriveParameterBlueprint :: TH.Name -> Set Purpose -> TH.ExpQ
+deriveParameterBlueprint tyName purpose = do
+  title <- lookupAnn @SchemaTitle tyName
+  description <- lookupAnn @SchemaDescription tyName
+  [| MkParameterBlueprint
+      { parameterTitle = listToMaybe (Text.pack . schemaTitleToString <$> title)
```

Comment:
Hang on, I though we agreed we were going to fail in general if we had several annotations? Again we are using `listToMaybe`?

Reply:
Good catch, this one slipped out of my attention (it was written before we agreed and git-stashed away).